src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -0,0 +1,90 @@
+package com.meilisearch.sdk;
+
+import com.google.gson.*;
+import com.google.gson.reflect.TypeToken;
+import com.meilisearch.sdk.exceptions.MeiliSearchApiException;
+
+/**
+ * Wrapper around MeilisearchHttpRequest class to use for MeiliSearch keys
+ *
+ * <p>Refer https://docs.meilisearch.com/reference/api/keys.html
+ */
+public class KeysHandler {
+    private final MeiliSearchHttpRequest meilisearchHttpRequest;
+    private final Gson gson =
+            new GsonBuilder().serializeNulls().setDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").create();
```

Comment:
Do we still need to configure the gson in the `toString()` one in the `Key.java`?

Reply:
I think so we use it in `createKey()`